bundles/org.openhab.binding.solarforecast/README.md

```diff
@@ -0,0 +1,346 @@
+# SolarForecast Binding
+
+This binding provides data from Solar Forecast services. 
+Use it to estimate your daily production, plan electric consumers like Electric Vehicle charging, heating or HVAC.
+Look ahead the next days in order to identify surplus / shortages in your energy planning.
+
+Supported Services
+
+- [Solcast](https://solcast.com/)
+    - Free [Hobbyist Plan](https://toolkit.solcast.com.au/register/hobbyist) with registration
+- [Forecast.Solar](https://forecast.solar/)
+    - Public, Personal and Professional [plans](https://forecast.solar/#accounts) available 
+
+Display Power values of Forecast and PV Inverter items
+
+<img src="./doc/SolcastPower.png" width="640" height="400"/>
+
+Display Energy values of Forecast and PV inverter items
+Yellow line shows *Daily Total Forecast*.
+
+<img src="./doc/SolcastCumulated.png" width="640" height="400"/>
+
+## Supported Things
+
+Each service needs one `xx-site` for your location and at least one photovoltaic `xx-plane`.  
+
+| Name                              | Thing Type ID |
+|-----------------------------------|---------------|
+| Solcast service site definition   | sc-site       |
+| Solcast PV Plane                  | sc-plane      |
+| Forecast Solar site location      | fs-site       |
+| Forecast Solar PV Plane           | fs-plane      |
+
+## Solcast Configuration
+
+[Solcast service](https://solcast.com/) requires a personal registration with an e-mail address.
+A free version for your personal home PV system is available in [Hobbyist Plan](https://toolkit.solcast.com.au/register/hobbyist)
+You need to configure your home photovoltaic system within the web interface.
+The `resourceId` for each PV plane is provided afterwards.
+
+In order to receive proper timestamps double check your time zone in *openHAB - Settings - Regional Settings*.
+Correct time zone is necessary to show correct forecast times in UI. 
+
+### Solcast Tuning
+
+You have the opportunity to [send your own measurements back to Solcast API](https://legacy-docs.solcast.com.au/#measurements-rooftop-site).
+This data is used internally to improve the forecast for your specific site.
+Configuration and channels can be set after checking the *Show advanced* checkbox.
+You need an item which reports the electric power for the specific rooftop. 
+If item isn't set, no measures will be sent.
+As described in [Solcast Rooftop Measurement](https://legacy-docs.solcast.com.au/#measurements-rooftop-site) check in beforehand if your measures are *sane*.
+
+- item is delivering correct values and they are stored in persistence
+- time zone setting in openHAB is correct to deliver correct timestamp
+
+After measurement is sent the `raw-tuning` channel is reporting the result.
+
```

Comment:
I believe this chapter is no longer relevant?

Reply:
True, removed